projects/services-ogc/src/lib/owc/types/owc-json.ts

```diff
@@ -338,7 +251,7 @@ export type LangString = string;
 
 
 export type WMS_Offering = 'http://www.opengis.net/spec/owc-geojson/1.0/req/wms' |
-    'http://schemas.opengis.net/wms/1.1.1' | 'http://schemas.opengis.net/wms/1.1.0';
+  'http://schemas.opengis.net/wms/1.1.1' | 'http://schemas.opengis.net/wms/1.1.0';
```

Comment:
This is a extension should be in `eoc-owc-json.ts`

Reply:
I've removed `'http://schemas.opengis.net/wms/1.1.1' | 'http://schemas.opengis.net/wms/1.1.0'` altogether from our type-definitions. The only purpose those strings had was encoding the WMS-version ... but that can also be inferred from the `operations/GetMap` request.